pkg/storaged/client.js

```diff
@@ -1156,15 +1157,20 @@
         proxy.TotalPhysicalUsed = [false, ""];
     if (!proxy.KeyDescription)
         proxy.KeyDescription = [false, ""];
-    stratis2_fetch_properties(proxy, ["TotalPhysicalSize", "TotalPhysicalUsed", "KeyDescription"]).then(values => {
-        if (values.TotalPhysicalSize)
-            proxy.TotalPhysicalSize = values.TotalPhysicalSize;
-        if (values.TotalPhysicalUsed)
-            proxy.TotalPhysicalUsed = [true, values.TotalPhysicalUsed];
-        if (values.KeyDescription)
-            proxy.KeyDescription = [true, values.KeyDescription];
-        client.update();
-    });
+    if (!proxy.ClevisInfo)
+        proxy.ClevisInfo = [false, ["", ""]];
+    stratis2_fetch_properties(proxy, ["TotalPhysicalSize", "TotalPhysicalUsed", "KeyDescription", "ClevisInfo"])
+            .then(values => {
+                if (values.TotalPhysicalSize)
+                    proxy.TotalPhysicalSize = values.TotalPhysicalSize;
+                if (values.TotalPhysicalUsed)
+                    proxy.TotalPhysicalUsed = [true, values.TotalPhysicalUsed];
+                if (values.KeyDescription)
+                    proxy.KeyDescription = [true, values.KeyDescription];
+                if (values.ClevisInfo)
+                    proxy.ClevisInfo = [true, values.ClevisInfo];
+                client.update();
```

Comment:
These 13 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230524-165921-2ee4ffa3-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)